### PR TITLE
Refactor insight package by splitting it into smaller ones

### PR DESCRIPTION
**Pull request by sanposhiho**
**What this PR does / why we need it**:
The responsibilities of the insight package have become too large.

So, split insight package into dto and insightstore

And milestonestore have no test. So, write small tests for `LoadMilestone`.
And I found a bug then and fixed it....(Sooooorrrry)

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/api/grpcapi/web_api.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 	"github.com/pipe-cd/pipe/pkg/crypto"
 	"github.com/pipe-cd/pipe/pkg/datastore"
 	"github.com/pipe-cd/pipe/pkg/git"
-	"github.com/pipe-cd/pipe/pkg/insight"
+	insightfilestore "github.com/pipe-cd/pipe/pkg/insight/insightstore"
```
Comment (nghialv) [inline pkg/app/api/grpcapi/web_api.go:41]: nit: Remove "insightfilestore".
I think "insightstore" is good enough and as I see we are using that name at other places too.

File: pkg/insight/dto/chunk.go [hunk 2]
```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package insight
+package dto
```
Comment (nghialv) [inline pkg/insight/dto/chunk.go:15]: @sanposhiho I understand why you named this as "dto". And as I see, this PR
- moves "store" code to "insight/insightstore" package
- moves "model" code to "insight/dto" package
- keeps utility functions in "insight" package

, right?

But how do you think about this
- moves "store" code to "insight/insightstore" package
- keeps "model" code and utility functions in "insight" package

I think this is simpler.

Comment (sanposhiho) [reply pkg/insight/dto/chunk.go]: Agree 👌 Moved files from dto to insight.

**Comment by sanposhiho** [comment]
@nghialv Thanks for your reviews, fixed them :D

**Comment by nghialv** [comment]
@sanposhiho Looks nice. Thank you for your work. 💯 
/approve

**Pull request merged**